api/CcsSso.Core.Api/Controllers/UserController.cs

```diff
@@ -38,7 +38,7 @@ public async Task Nominate([FromBody] string email)
     [HttpPost("activation-emails")]
     [Consumes("application/x-www-form-urlencoded")]
     [SwaggerOperation(Tags = new[] { "User" })]
-    public async Task SendUserActivationEmail(IFormCollection userDetails, [FromQuery(Name = "is-expired")]bool isExpired = false)
+    public async Task SendUserActivationEmail(IFormCollection userDetails, [FromQuery(Name = "is-expired")]bool isExpired = true)
```

Comment:
Changing the default value of the request parameter will make the behaviour of all request using this endpoint different. Therefore need to test all of them do the changes in UI or you need to check the logic.